acbuild/acbuild.go

```diff
@@ -153,7 +153,7 @@ func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd
 
 		cmdExitCode = cf(cmd, args)
 
-		err = a.Write(aciToModify, true, false, nil)
+		err = a.Write(aciToModify, true, false, false, nil)
```

Comment:
I actually think it would make more sense for it to compress the image in this case. Thoughts, @jonboulle?